packages/osd-optimizer/src/worker/webpack.config.ts

```diff
@@ -204,7 +204,9 @@ export function getWebpackConfig(bundle: Bundle, bundleRefs: BundleRefs, worker:
         },
         {
           test: /\.(js|tsx?)$/,
-          exclude: /node_modules/,
+          // vega-lite and some of its dependencies don't have es5 builds
+          // so we need to build from source and transpile for webpack v4
+          exclude: /[\/\\]node_modules[\/\\](?!vega-(lite-next|label|functions)[\/\\])/,
```

Comment:
Have we checked that this doesn't interfere with the original vega-lite?

Reply:
Yeah, I can confirm it doesn't in a couple ways - when I was creating https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3076 from this diff, I accidentally validated that. The jest tests do their own direct import of `vega-lite` and I had initially missed changing them, because they all still worked. And without the `lite-next` part of that regex correctly ensures that `vega-lite` will be ignored.